modules/tracktion_engine/playback/graph/tracktion_TracktionNodePlayer.h

```diff
@@ -111,14 +111,14 @@ class TracktionNodePlayer
     /** @internal */
     void enablePooledMemoryAllocations (bool enablePooledMemory)
     {
-        nodePlayer.enablePooledMemoryAllocations (enablePooledMemory);
+        //dddnodePlayer.enablePooledMemoryAllocations (enablePooledMemory);
     }
     
 private:
     tracktion::graph::PlayHeadState& playHeadState;
     ProcessState& processState;
     MidiMessageArray scratchMidi;
-    tracktion::graph::LockFreeMultiThreadedNodePlayer nodePlayer;
```

Comment:
Interesting. Is there any worry that using a locking player would negatively affect performance in some way?

Reply:
Just been testing on release on my device and I'm still able to comfortably run 10+ magic tunes (which include the reverb).